libsysbox/syscont/spec.go

```diff
@@ -607,6 +610,8 @@ func cfgSysboxFsMounts(spec *specs.Spec, sysFs *sysbox.Fs) {
 			)
 	}
 
+	SysboxFsDir = sysFs.Mountpoint
```

Comment:
Why are you updating this global here? I don't see any code (other than the global-vars) referencing this variable, hence the question ...

Reply:
There is another piece of code referencing it in rootfs_linux.go:

```golang
1149 │ func needUidShiftOnBindSrc(mount *configs.Mount, config *configs.Config) (bool, error) {                                                                                                                                                                                                                               
1150 │                                                                                                                                                                                                                                                                                                                        
1151 │    // sysbox-fs handles uid(gid) shifting itself, so no need for mounting shiftfs on top                                                                                                                                                                                                                               
1152 │    if strings.HasPrefix(mount.Source, syscont.SysboxFsDir+"/") {                                                                                                                                                                                                                                                       
1153 │       return false, nil                                                                                                                                                                                                                                                                                                
1154 │    }   
```

This is why I opted for updating the global var.